packages/esm-appointments-app/src/patient-appointments/patient-appointments-header.scss

```diff
@@ -0,0 +1,37 @@
+@use '@carbon/styles/scss/type';
+@use '@carbon/colors';
+@use '@carbon/styles/scss/spacing';
+
+.titleContainer {
```

Comment:
Looks like the DOM element that uses this only has a button; so all this is probably not needed.